packages/api/package.json

```diff
@@ -44,6 +43,7 @@
     "express": "^4.17.1",
     "node-cache": "^5.1.2",
     "node-fetch": "^2.6.6",
+    "prisma-docs-generator": "^0.5.0",
```

Comment:
^ this should be a dev dependency

Reply:
Strangely enough - leaving the doc generator in the dev dependency breaks the initialization of the docker containers. This is because the generator is used at compile time from the schema.prisma file. 
Not sure if there is a way to get around this.